infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -252,6 +252,67 @@ mode, no need to publish).
 3. Copy the JWT Token, it will be used later when creating the Zoom cloud function.
 
 4. Activate the app
+EOT
+    }
+  }
+  oauth_long_refreshToken_connectors = {
+    dropbox-business = {
+      source_kind : "dropbox-business",
+      display_name : "Dropbox Business"
+      example_api_calls : [
+        "/2/team/members/list_v2",
```

Comment:
These endpoints are at the moment useless, as we need support in the test tool to include POST requests with body